tailcfg/tailcfg.go

```diff
@@ -694,6 +695,17 @@ type FilterRule struct {
 	// DstPorts are the port ranges to allow once a source IP
 	// matches (is in the CIDR described by SrcIPs & SrcBits).
 	DstPorts []NetPortRange
+
+	// IPProto is the IP protocol number to match.
+	// As a special case, zero matches both TCP (6) and UDP (17).
+	//
+	// Numbers outside the uint8 range (below 0 or above 255) are
+	// reserved for Tailscale's use.
+	//
+	// To actually specify zero, use 256.
```

Comment:
I'd not bother with this. IP protocol zero is "IPv6 hop-by-hop options", which I sincerely hope we'll never attempt.

Reply:
Removed.